src/api/forms/FormsAPI.js

```diff
@@ -158,7 +159,8 @@ export default class FormsAPI extends EventEmitter {
                     key = property.join('.');
                 }
 
-                changes[key] = data.value;
+                _.set(changes, `${key}`, data.value);
+                self.emit('formPropertiesChanged', changes);
```

Comment:
What's the difference between this and the `onChange` callback passed into this function?